src/test/run-make/issue-26006/Makefile

```diff
@@ -0,0 +1,16 @@
+-include ../tools.mk
+
+ifndef IS_WINDOWS
+all: time
+
+time: libc
+	mkdir -p out/time out/time/deps
+	ln -sf out/libc/liblibc.rlib out/time/deps/
+	$(RUSTC) in/time/lib.rs -Ldependency=out/time/deps/
```

Comment:
Isn't this command expected to fail without an ICE error message? (e.g, shouldn't there be a piping or assertion of some form here?)